apollo-runtime-kotlin/src/commonMain/kotlin/com/apollographql/apollo/ApolloClient.kt

```diff
@@ -0,0 +1,41 @@
+package com.apollographql.apollo
+
+import com.apollographql.apollo.api.ApolloExperimental
+import com.apollographql.apollo.api.ExecutionContext
+import com.apollographql.apollo.api.Mutation
+import com.apollographql.apollo.api.Operation
+import com.apollographql.apollo.api.Query
+import com.apollographql.apollo.api.ScalarTypeAdapters
+import com.apollographql.apollo.executor.NetworkExecutor
+import com.apollographql.apollo.executor.RequestExecutor
+import com.apollographql.apollo.internal.RealApolloCall
+import com.apollographql.apollo.network.NetworkTransport
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+
+@ApolloExperimental
+@ExperimentalCoroutinesApi
```

Comment:
Some users might not want to enable `ExperimentalCoroutinesApi` in their repo. As long as it's only for multiplatform I guess it's fine but the day `apollo-runtime-kotlin` becomes the default JVM implementation, it might be an issue (if the api hasn't been promoted to stable before that)

Reply:
Well, flow operators are still marked with `flatMapLatest`, so no sure what we do with this?

Reply:
I guess it means keeping the JVM runtime around long enough for these flow operators to become stable?

Reply:
yeah, we should keep JVM for some time.